glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -97,12 +97,47 @@ open class GleanInternalAPI internal constructor () {
         }
     }
 
+    /**
+     * Enable or disable Glean collection and upload.
+     *
+     * Metric collection is enabled by default.
+     *
+     * When uploading is disabled, metrics aren't recorded at all and no data
+     * is uploaded.
+     *
+     * When disabling, all pending metrics, events and queued pings are cleared.
+     *
+     * When enabling, the core Glean metrics are recreated.
+     *
+     * @param enabled When true, enable metric collection.
+     */
+    fun setUploadEnabled(enabled: Boolean) {
+        // logger.info("Metrics enabled: $enabled")
+        // val origUploadEnabled = uploadEnabled
+        // uploadEnabled = enabled
+        // if (isInitialized() && origUploadEnabled != enabled) {
+        //     onChangeUploadEnabled(enabled)
+        // }
+        // TODO: stub
```

Comment:
Can we have stubbed methods log (or assert?). I was thinking about something like `"setUploadEnabled not implemented yet"`. This would help us not forget about these :)